nerTagger.py

```diff
@@ -422,6 +531,10 @@ def annotate(output_format,
 
 
 if __name__ == "__main__":
+
+    architectures = ['BidLSTM_CRF', 'BidLSTM_CNN_CRF', 'BidLSTM_CNN_CRF', 'BidGRU_CRF', 'BidLSTM_CNN', 'BidLSTM_CRF_CASING', 
+                     'bert-base-en', 'bert-base-en', 'scibert', 'biobert']
```

Comment:
I noticed there are two 'bert-base-en'

Reply:
oups the second one should be `bert-large-en`!